packages/slides-frontend/src/utils/helperFunctions.js

```diff
@@ -0,0 +1,33 @@
+
+const canDeleteImageFromBackend = (imageName, slidesStringified) => {
+    const slides = JSON.parse(slidesStringified);
```

Comment:
Why don't you pass `slides` array of objects directly? There is no point now in using `slidesStringified`